web/src/search/index.tsx

```diff
@@ -73,6 +72,12 @@ export function parseSearchURLPatternType(query: string): SearchPatternType | un
     return patternType
 }
 
+export function parseSearchURLCaseSensitive(query: string): boolean {
```

Comment:
The current name makes me think that this is a function for parsing a search query accounting for case-sensitivity.

```suggestion
/**
 * Returns `true` if `case=yes` is set in URLSearchParams.
 */
export function searchURLIsCaseSensitive(query: string): boolean {
```